lib/pymedphys/_mosaiq/delivery.py

```diff
@@ -281,36 +286,61 @@ def delivery_data_sql(connection, field_id):
             TxFieldPoint.Coll_Ang,
             TxFieldPoint.Coll_Y1,
             TxFieldPoint.Coll_Y2
+            {',TxFieldPoint.RowVers' if include_rowvers else ''}
```

Comment:
The above comment about being able to return `RowVers` all the time should be able to simplify things here :slightly_smiling_face: